dask_ml/datasets.py

```diff
@@ -451,8 +451,8 @@ def make_classification_df(
             [
                 X_df,
                 dd.from_array(
-                    np.array([random_date(*dates)] * len(X_df)),
-                    chunksize=chunks,
+                    np.array([random_date(*dates) for i in range(len(X_df))]),
```

Comment:
Looks like there's some test errors because `random_dates` doesn't have a random seed. Maybe this signature would help?

```
def random_dates(start, end, random_state=None):
    rng = check_random_state(random_state)
    ...
```

Reply:
Yeah, that could be a probable reason, I'll work on it. And also, the test written for the function(`make_classification_df`) doesn't check for true randomness. I'm thinking of adding a new test/modifying the existing one for the same, will that suffice?
link to [test_make_classification_df](https://github.com/dask/dask-ml/blob/80eb0d81b22df3762706384c3ca1c0be4137e591/tests/test_datasets.py#L67)

Reply:
Isn't #845 filed around having only one unique value in the date column? I'd make sure there's at least a couple values.